src/plugins/contextMenu/menu.js

```diff
@@ -265,6 +265,7 @@ class Menu {
     this.close();
     this.parentMenu = null;
     this.eventManager.destroy();
+    this.container.remove();
```

Comment:
`.remove()` method is not supported by IE (https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove). Additionally, can I ask you to add/update test which will test if that element was removed?